### ISSUE TITLE
Is learning rate decay a regularization technique?

### ISSUE THREAD

**Issue by shuaibaslam2019** [pytorch/pytorch#51899]
Upto my understanding, it is a regularization technique, because it helps to learn model correctly and in generalization. But I am still confused at whether it would be correct or not to call it a "regularization" method.??

Thank you!

**Comment by izdeby** [comment]
Please use our [forum](https://discuss.pytorch.org/) for questions

**Comment by mrshenli** [comment]
Hey @shuaibaslam2019, as @izdeby suggested above, please use forum for questions. I am closing this issue for now. Please feel free to open another issue if this turns out to be a bug report or feature request. Thanks!
